### DISCUSSION TITLE
BITCOIN CORE GUIDE [MACBOOK USER EDITION]

### DISCUSSION THREAD

**Discussion by joeatang** [ordinals/ord#1608] [Q&A] [answered]
Requesting an ord x bitcoin core user guide. From this basic research ive done for 2 weeks, it seems as if Macbook users do not get 'bitcoind' or bitcoin-cli' 

I'm not that techy so ive been stuck on these steps for a couple of weeks trying a bunch of different options from chatgpt and other get hub forums. non seem to be a solution because im still here. This would be greatly appreciated!

**Comment by casey** [comment, marked as answer]
I think a simple mac guide would be great. In particular, bitcoin can be installed with `brew install bitcoin` and run with `brew services start bitcoin`. If we can get ord to homebrew, so users can `brew install ord`, that would be a huge win.

**Reply by casey** [reply]
Made this issue to track packaging in general: https://github.com/casey/ord/issues/1609

**Reply by joeatang** [reply]
HI Casey! Congrats on the journey so far bro. Been here since the beginning. Im a mac user thats now struggling to run ord on my macbook pro:

have repeatedly received this error message (mac book oro user and have tried to set up core the normal way and using @ETS guide) 

Error Message from Debug.Log: 
ERROR: ReadBlockFromDisk: Errors in block header at FlatFilePos(nFile=19, nPos=4590870)
2023-12-17T02:32:39Z  Failed to read block
2023-12-17T02:32:39Z Error: A fatal internal error occurred, see debug.log for details
2023-12-17T02:33:13Z ERROR: ProcessNewBlock: ActivateBestChain failed (Failed to read block)
2023-12-17T02:33:13Z ERROR: ReadBlockFromDisk: Errors in block header at FlatFilePos(nFile=19, nPos=4590870)
2023-12-17T02:33:13Z  Failed to read block
2023-12-17T02:33:13Z Error: A fatal internal error occurred, see debug.log for details
2023-12-17T02:33:13Z tor: Thread interrupt
2023-12-17T02:33:13Z Shutdown: In progress...
2023-12-17T02:33:13Z torcontrol thread exit
2023-12-17T02:33:13Z opencon thread exit
2023-12-17T02:33:13Z addcon thread exit

What ive tried: 
I basically factory reset my mac and have a new SSD 1TB , yet this error message continue to pop up. Im stumped and need a smater person to help me figure this out! 

If you can direct any of your resources/peeps my way this would be awesome

**Reply by casey** [reply]
This quite strange. It looks like a core error, as far as I can tell it's unrelated to ord, so I don't have any clever ideas for how to fix it.

**Comment by sanielo** [comment]
Where can we get the macbook user guide. I run the bitcoin core and it's updated but I am stuck in the installations and sync of ord. 
Please help, I am that techy too. Stuck since days

**Reply by AdmiralOrange** [reply]
Yeah me too - Ord doesn't install with the command from Casey's docs "curl --proto '=https' --tlsv1.2 -fsLS https://ordinals.com/install.sh | bash -s"

**Comment by AdmiralOrange** [comment]
I am no dev - but proud finally running a full node on an iMAC. Now stuck in installations of Ord - in Terminal "curl --proto '=https' --tlsv1.2 -fsLS https://ordinals.com/install.sh | bash -s" works somehow and installs "Ord" in "Bin", but when I try "ord --version" Terminal says  "command not found". So I guess Ord is not installed. Are there issues with iMAC? I realize installations mention Darwin - a development Apple stopped. Maybe this is the reason, or some shell-settings necessary or other petty little stuff or is there another way to install ORD on iMAC?

**Reply by farhad-zada** [reply]
I am not sure if this is your problem but you can try 


- add 
```bash
export PATH="$PATH:/usr/local/bin/ord"
```
- the this command
```bash
source ~/.zshrc
```
if you are using bash try `.bash_profile`
